deploy/docker-compose-CeleryExecutor.yml

```diff
@@ -60,6 +60,39 @@ services:
         ports:
             - "5432:5432"
 
+    postgres-exporter:
+        image: quay.io/prometheuscommunity/postgres-exporter
+        environment:
+            DATA_SOURCE_URI: postgres?sslmode=disable
+            DATA_SOURCE_USER: ${POSTGRES_USER}
+            DATA_SOURCE_PASS: ${POSTGRES_PASSWORD}
+
+    statsd-exporter:
+        image: ranlu/statsd-exporter
+        command: "--statsd.listen-udp=:9125 --web.listen-address=:9102 --statsd.mapping-config=/statsd.conf"
+        ports:
+            - "9102:9102"
+            - "9125:9125/udp"
+
+    prometheus:
+        image: ranlu/prometheus
```

Comment:
Sure, I want to put them in a separate repo though, since the changes to these images are not so tightly related seuron, (for example, add metrics in corgie or synaptor to measure performance and visualize them in grafana). `seuron_deployment` would be a appropriate place if it is not taken already